packages/flutter/test/material/text_field_test.dart

```diff
@@ -6086,6 +6086,45 @@ void main() {
     semantics.dispose();
   });
 
+  // Regressing test for https://github.com/flutter/flutter/issues/99763
+  testWidgets('Update textField semantics when obscureText changes', (WidgetTester tester) async {
```

Comment:
This test seems to pass even without the change. It may not capture the right situation?

Reply:
Sorry, updated test, it should capture the change now